styles/SocialMediaPhotos.module.scss

```diff
@@ -0,0 +1,32 @@
+.container {
+    display: grid;
+    padding: 0 $grid;
+    grid-template-columns: 1fr 1fr;
+
+    @include media-query(medium) {
+        grid-template-columns: repeat(3, 1fr);
+    }
+
+    @include media-query(large) {
+        padding: 0 $grid * 2;
+        grid-template-columns: repeat(4, 1fr);
+    }
+}
+
+.image {
+    list-style-type: none;
+    border: 2px solid var(--color--white);
+
+    /* Hide the last two images on mobile */
```

Comment:
This looks good, I thought I'd mention this is also something we could do with a [useMedia](https://usehooks.com/useMedia/) hook if you wanted to explore using that instead. I'm not 100% sure if there would be any benifits to this or not, just thought I could be helpful to know

Reply:
useMedia won't work without some clever tweaking - the window isn't available in Next.js as it's running server side. I tried to use the following code to resolve this, but I made no headway with it. Not worth the time investment - [window not defined with next.js](https://dev.to/vvo/how-to-solve-window-is-not-defined-errors-in-react-and-next-js-5f97)

Reply:
I will commit the hook as I think it might be useful in the future

Reply:
Thanks for this James, definitely useful to know!